frontend/components/CourseActions/components/CourseDescriptionAndStats.js

```diff
@@ -110,9 +110,10 @@ class CourseDescriptionAndStats extends React.Component {
             <i className="fa fa-users"/>,
             <StatsModal
               toggler={
-                <a>
-                  <div><span className="number">{this.props.courseDto.learners.length}</span> students</div>
-                </a> 
+                <button type="button" className="button students-stats-button">
```

Comment:
We have a special class for buttons that need no styling `button -clear`.

Reply:
So how does the user know "N students" is clickable if it won't display a hover effect like the ones defined in `focus-styles.css`?

Reply:
Done!

Reply:
Superb, so both PRs are ready, I'll review both of them tomorrow!